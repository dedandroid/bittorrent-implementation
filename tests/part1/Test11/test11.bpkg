ident:37d592b9bfaca09e9152f3580c10a48eb5347ed2d8e787629b48e90e61f057e1c253036eb453cb5579d53e49f395a866ab9aef894ed09250c26f0a80d1579bd366e566ea4baeefea1492e12b272c2ff86ddc4c687065f5f09828943cb68d576b5379fd17d8ccdcd64fee42af82dd939e6076e9db198e554a4388528d45ad83ce4342c1ddb5b0a0af32784f584f6c32b75f9216fccbc6b75e9d6ecb61acdde8448d694550016b8c929f05a6653421c7544bd812814fdcb6e46e9050584890ff53a2cb54d93a5e04262c77ccf04903857283ed7b6b5b95fcc28394f944247aaec21af9554a0d009c21fb5e4926ea088daa793de77e47ee30f2c510e37cd7464e0c7395a04e82cb3cef1fff26c0e062f6e697c380102eb5a94b84aba6b871b679c01839849b240ce57692138ccfd7540241a7a3c3ee8eb632cc4e0dacc832c3415e8f143f3bee110de5be26aeed0a04b53454884b32943a11fdf139f160b747c7b1c390fc28052271f62202362ed65ae0ba45a31cb21e48f3f25f485663ccdac840de3faf1cd54c83ed22588bb578f4044e28dd7e943d0b1f831dba86f0ee4f28d50a278bbb8c69bedcb863f53d47c609b03d8b8361fcaaa7750d9f3cc749d429466c1f080f4aaf2142ed11ad8e6629f6f53043842dfcc1d03ce4d81668c81be0010549976832912cd00a820eac6b75b86bdb462af5c86f53f5e87e61bdc227a83
filename:test11.data
size:512
nhashes:3
hashes:
	f8d956968fcf218b80611056aac85316a4be71df3843901aa27f3c49d1a4a12b
	808d13d9ed9f9d7242ca5a95f1d879c734f88511360cf905b5fa6a4e4d0dbea4
	94bf0b2c7b798ff7662fb8acee1656aaa2234478974b7358f52bc6023879aec3
nchunks:4
chunks:
	943586f509019a99d5e03b50525137d8a9f1514221abc4dba019db41f0824e95,0,128
	4f2c3c22ca76f11e73f78fbf50f00e213936d219b21bd0ed698d7d9832e312a3,128,128
	17ff8bf5d056708c3eb1e0dd94cb36fab13eb1b3ec8d0c1ccdf185110d31814b,256,128
	83aa3b884534d7915b937fd2c9ade56221c4e958be5e37c00a0f7afaeae7779e,384,128
