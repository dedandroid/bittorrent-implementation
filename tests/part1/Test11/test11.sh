i='11'
../pkgchecker test$i.bpkg -hashes_of 808d13d9ed9f9d7242ca5a95f1d879c734f88511360cf905b5fa6a4e4d0dbea4 > test$i.out 

#Surpress output for not
diff test$i.out test$i.expected > /dev/null

if [ $? -eq 0 ]; then
        echo "Output matches"
else
        echo "Outputs do NOT match!"
fi

