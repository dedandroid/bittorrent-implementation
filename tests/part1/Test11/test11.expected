943586f509019a99d5e03b50525137d8a9f1514221abc4dba019db41f0824e95
4f2c3c22ca76f11e73f78fbf50f00e213936d219b21bd0ed698d7d9832e312a3
